server/app/controller/chat/history_controller.py

```diff
@@ -3,7 +3,7 @@
 from fastapi_pagination.ext.sqlmodel import paginate
 from app.model.chat.chat_history import ChatHistoryOut, ChatHistoryIn, ChatHistory, ChatHistoryUpdate
 from fastapi_babel import _
-from sqlmodel import Session, select, desc
+from sqlmodel import Session, select, desc, asc, case
```

Comment:
`asc` is imported but not used

```suggestion
from sqlmodel import Session, select, desc, case
```